zomes/hc_zome_trust_atom/src/trust_atom.rs

```diff
@@ -141,8 +141,17 @@ fn normalize_value(value_str: Option<String>) -> ExternResult<Option<String>> {
 
 fn create_link_tag(link_direction: &LinkDirection, chunk_options: &[Option<String>]) -> LinkTag {
   let mut chunks: Vec<String> = vec![];
+  if let Some(content) = chunk_options[0] {
```

Comment:
The first chunk is not always content, notably in `query` methods -- ie this fn is used by both create and query.  Been thinking to split it into two variants ...

Reply:
if chunk is None we could push empty string ""

see branch zome/trust-graph

Reply:
fixed to correct amount of nul bytes